### ISSUE TITLE
More efficient slicing of arrays in tax.py

### ISSUE THREAD

**Issue by jdebacker** [open-source-economics/OG-USA#298]
Tax.py has many lines of code to take slices from differently shaped arrays.  These can be replaced by using the ellipsis method on the numpy arrays.  E.g. 

```
    A = etr_params[...,0]
    B = etr_params[...,1]
    C = etr_params[...,2]
    D = etr_params[...,3]
    max_x = etr_params[...,4]
    min_x = etr_params[...,5]
    max_y = etr_params[...,6]
    min_y = etr_params[...,7]
    shift_x = etr_params[...,8]
    shift_y = etr_params[...,9]
    shift = etr_params[...,10]
    share = etr_params[...,11]
```

**Comment by hdoupe** [comment]
@jdebacker This was closed by #343, right?

**Comment by jdebacker** [comment]
Yes.  Closed by PR #343
